docs/src/sciml_style.md

```diff
@@ -30,3 +30,28 @@ Or to use `SciMLStyle` except change one of the settings:
 ```julia
 format("file.jl", SciMLStyle(), remove_extra_newlines=false)
 ```
+
+## Additional Options
+
+The `SciMLStyle` supports the additional option `variable_dict_indent`,
+which is set to `false` by default.
+It allows `Dict` definitions without aligning to the opening parenthesis:
```

Comment:
since you've added this functionality to the YAS functions, which SciML fallbacks too we should mention this works for YAS as well